README.md

```diff
@@ -1213,6 +1214,23 @@ ardrive list-drive -d { drive id from create drive }
 ardrive download-file -f { file id from upload file }
 ```
 
+### Persistent Caching of ArFS Entity Metadata<a id="metadata-caching"></a>
+
+To avoid redundant requests to the Arweave network for immutable ArFS entity metadata, a persistent file cache is created and maintained at:
+
+```
+Windows: <os.homedir()>/ardrive-caches/metadata
+Non-Windows: <os.homedir()>/.ardrive/caches/metadata
+```
+
+The `XDG_CACHE_HOME` environment variable is honored, where applicable, and will be used in place of `os.homedir()` in the scenarios described above.
+
+Metadata cache logging to stderr can be enabled by setting the `ARDRIVE_CACHE_LOG` environment variable to `1`.
+
+Cache performance is UNDEFINED for multi-process scenarios, but is presumed to be generally usable.
+
+The cache can be manually cleared safely at any time that any integrating app is not in operation.
```

Comment:
Does it worth adding a command `clean-cache` for this case?

Reply:
Will consider adding that in the future as the cache's uses expand.